packages/yarnpkg-builder/package.json

```diff
@@ -13,7 +13,7 @@
     "babel-loader": "^8.1.0",
     "chalk": "^3.0.0",
     "clipanion": "^3.0.0-rc.10",
-    "esbuild-wasm": "^0.8.36",
+    "esbuild-wasm": "^0.11.20",
     "filesize": "^4.1.2",
     "fork-ts-checker-webpack-plugin": "^5.0.0",
```

Comment:
Should be able to remove the webpack related dependencies now

Reply:
Never mind, it's still in use by `makeConfig` which `@yarnpkg/pnp` uses to build itself